api-extractor.json

```diff
@@ -0,0 +1,411 @@
+/**
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30365&commitId=0c5fac8811f39a845ade1ff321f41f1d56bd7a7a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Shimmer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.Data loaded.default.chromium.png|0| Added|
|Shimmer.Data not loaded.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>